go/chat/sender.go

```diff
@@ -1160,6 +1160,18 @@ func (s *Deliverer) processAttachment(ctx context.Context, obr chat1.OutboxRecor
 	return obr, nil
 }
 
+type unfurlerPermError struct{}
+
+func (e unfurlerPermError) Error() string {
+	return "unfurler permanent error"
```

Comment:
can you store the original error so we can make sure it's not on our end?

Reply:
Sure

Reply:
Oh actually we don't have it here, we can easily see it from the logs though.